zds/utils/templatetags/interventions.py

```diff
@@ -182,3 +183,11 @@ def alerts_list(user):
                           'text': alert.text})
 
     return {'alerts': total, 'nb_alerts': nb_alerts}
+
+
+@register.filter(name='waiting_count')
+def waiting_count(content_type):
+    return Validation.objects.filter(
```

Comment:
Je vais faire mon chiant, et demander une petite levée d'erreur si `content_type not in zds.tutorialsv2.models.
TYPE_CHOICES_DICT`. Pour être sur :)